src/assets/wise5/authoringTool/node/edit-node-title/edit-node-title.component.ts

```diff
@@ -4,21 +4,23 @@ import { TeacherProjectService } from '../../../services/teacherProjectService';
 
 @Component({
   selector: 'edit-node-title',
-  templateUrl: './edit-node-title.component.html',
-  styleUrls: ['./edit-node-title.component.scss']
+  templateUrl: './edit-node-title.component.html'
 })
 export class EditNodeTitleComponent {
   protected isGroupNode: boolean;
+  protected label: string;
   @Input() node: Node;
   protected nodeJson: any;
-  protected nodePosition: string;
 
   constructor(private projectService: TeacherProjectService) {}
 
   ngOnChanges(): void {
-    this.isGroupNode = this.node.isGroup();
     this.nodeJson = this.projectService.getNodeById(this.node.id);
-    this.nodePosition = this.projectService.getNodePositionById(this.node.id);
+    this.isGroupNode = this.node.isGroup();
+    this.label =
+      (this.isGroupNode ? $localize`Lesson Title` : $localize`Step Title`) +
+      ' ' +
+      this.projectService.getNodePositionById(this.node.id);
```

Comment:
Remove isGroupNode class variable and use this.node.isGroup() directly in the ternary expression?
```suggestion
    this.label =
      (this.node.isGroup() ? $localize`Lesson Title` : $localize`Step Title`) +
      ' ' +
      this.projectService.getNodePositionById(this.node.id);
```